ansible/configs/osp-migration/post_infra.yml

```diff
@@ -17,13 +17,26 @@
         name: bastion-student-user
       when: install_student_user | bool
 
+    - name: Check if open-ini exists
+      stat:
+        path: /usr/local/bin/open-init.sh
+      register: open_init
+      when: install_bastion | bool
+
+    - name: Execute open-init script on bastion
+      shell: "/usr/local/bin/open-init.sh {{ student_name }}"
```

Comment:
you're not using any feature from the `shell` module (pipes, redirections, variables, ..)

you can use `command` instead of `shell`

Reply:
done